addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -387,6 +387,11 @@ registerModel({
         menu_id: attr({
             default: null,
         }),
+        mobileMessagingNavbar: one2one('mail.mobile_messaging_navbar', {
+            default: insertAndReplace(),
+            inverse: 'discuss',
+            isCausal: true,
+        }),
```

Comment:
I would be expecting no default, and only create when in mobile.

Also should probably be suffixed `View`